apps/widget-configurator/src/app/embedDialog/utils/sanitizeParameters.ts

```diff
@@ -0,0 +1,10 @@
+import { CowSwapWidgetParams } from '@cowprotocol/widget-lib'
+
+export function sanitizeParameters(params: CowSwapWidgetParams) {
+  return {
+    ...params,
+    provider: `<eip-1193 provider>`,
+    '//': 'Please contact https://cowprotocol.typeform.com/to/rONXaxHV to enable your partner fee.',
```

Comment:
Isn't there any other way to add the comment? the code is JS

Reply:
<img width="815" alt="image" src="https://github.com/cowprotocol/cowswap/assets/7122625/9018b4b8-2ced-4994-9f7e-8776928fa4e5">

Fixed

Reply:
I made a PR on top. If you agree with that one, we can merge this 2 PRs: https://github.com/cowprotocol/cowswap/pull/3345